daemon/daemon.go

```diff
@@ -404,6 +404,9 @@ func (daemon *Daemon) restore() error {
 							c.Unlock()
 						}
 					}
+				} else if !c.IsPaused() && alive {
```

Comment:
We should clean up this if clause.
Similar check got us into trouble one if-block up already.

Reply:
Probably a `switch` statement will be more resilient to future changes.

Reply:
Done